app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    if params[:search] == "actor" && params[:actor_id].present? 
+      actor_id = params[:actor_id]
+      @search_actors = Actor.where("id::text LIKE ?", "%#{actor_id}%")
```

Comment:
This would be an MVC infraction because of the use of `.where` - this should be in the model and tested.